chapter2/tests/t20/terraform/base/efs.tf

```diff
@@ -0,0 +1,23 @@
+module "efs" {
+  source           = "terraform-aws-modules/efs/aws"
+  version          = "1.1.1"
+  name             = local.labels.efs
+  performance_mode = "generalPurpose"
```

Comment:
don't hardcode. They may change between prod and staging

Reply:
> don't hardcode. They may change between prod and staging

@dmitry-mightydevops 
Done